app/bundles/MessengerBundle/Translations/en_US/validators.ini

```diff
@@ -0,0 +1,2 @@
+mautic.messenger.dsn.invalid_dsn="Invalid DSN. Please make sure you entered all the needed fields."
+mautic.messenger.dsn.unsupported_scheme="Unsupported scheme."
```

Comment:
What does this actually mean to the user? Can we provide them with some more useful feedback (for example, what they need to do to correct the error, and where to look for support)?

Reply:
Addressed in d43d679.